src/framework/_breakpoints.scss

```diff
@@ -13,6 +13,16 @@ $breakpoints: (
   'xxl': 1920,
 ) !default;
 
+$breakpoints-height: (
+  'xxs': 0,
+  'xs': 360,
+  's': 576,
+  'm': 768,
+  'l': 960,
+  'xl': 1080,
+  'xxl': 1440,
+) !default;
+
```

Comment:
Should we add "standard" device height in this list ? 
for exemple `ipad-landscape` or `ipad-portrait`

Reply:
I think we should not add theses in default breakpoints. We didn't do for width breakpoints because this project should not be specific but must be expandable.